### DISCUSSION TITLE
Delete Chat prompt box misaligned on mobile device.

### DISCUSSION THREAD

**Discussion by parthpat12** [danny-avila/LibreChat#4930] [Feature Requests & Suggestions]
### What happened?

The prompt to delete a chat needs to have text and button center aligned. Currently it’s moved to the left while keeping title in the center. 

Tested on iPhone 15 Pro using Safari
![LibreChat jpeg](https://github.com/user-attachments/assets/4ec1b00e-d5aa-422f-94cb-f37ed186c7b3)

### Steps to Reproduce

1. Select one of the conversation
2. Select Delete

### What browsers are you seeing the problem on?

Mobile (iOS)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This is not a bug rather a suggestion, which I think you're asking for an alignment with chatgpt. @berry-13 can comment if he agrees

![firefox_lcazwfibPv](https://github.com/user-attachments/assets/9065dbd0-a9cf-4e6a-9078-f2ce33ad23f7)

**Reply by berry-13** [reply]
yes, I can work on it

**Reply by berry-13** [reply]
@parthpat12 updated here #5011
